tensorbay/client/segment.py

```diff
@@ -653,6 +680,51 @@ def list_data_paths(self) -> PagingList[str]:
         """
         return PagingList(self._generate_data_paths, 128)
 
+    def get_data(self, remote_path: str) -> RemoteData:
+        """Get required Data object from a dataset segment.
+
+        Arguments:
+            remote_path: The remote paths of the required data.
+
+        Returns:
+            :class:`~tensorbay.dataset.data.RemoteData`.
+
+        Raises:
+            ResourceNotExistError: When the required data does not exist.
+
+        """
+        if remote_path not in self.list_data_paths():
+            raise ResourceNotExistError(resource="data", identification=remote_path)
+
+        response = self._get_data_details(remote_path)
+        data_details = response["dataDetails"][0]
+
+        mask_urls = {}
+        for key in _MASK_KEYS:
+            mask_urls[key] = self._get_mask_url(key.upper(), remote_path)
```

Comment:
If the mask url is get from another API (_get_mask_url), there will be no reason to get it here, get url lazily (request url just when user need it) will be a better design.

But, here we should get mask url in `_get_data_detail` or `_get_label` in order to decrease the request count for better performance.